app/src/main/java/org/fossasia/openevent/general/event/location/EventLocation.kt

```diff
@@ -0,0 +1,13 @@
+package org.fossasia.openevent.general.event.location
+
+import com.github.jasminb.jsonapi.LongIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Type
+
+@Type("event-location")
+data class EventLocation(
+    @Id(LongIdHandler::class)
+    val id: Long?,
+    val name: String?,
+    val slug: String?
```

Comment:
Why are these nullable?

Reply:
I can intialize them with empty string also i just followed the pattern which was found in most of the data classes

Reply:
@iamareebjamal should i initialize with them empty string instead of being nullable type?

Reply:
No, make them non nullable without intializers. Tell me if they don't work